### DISCUSSION TITLE
Unable to build Docker

### DISCUSSION THREAD

**Discussion by tricaricom** [neonbjb/tortoise-tts#839] [Q&A] [answered]
Hello,

I am trying to build for Docker per the [instructions ](https://github.com/neonbjb/tortoise-tts?tab=readme-ov-file#docker)and I am getting this error:

`error: tokenizers 0.14.0 is installed but tokenizers!=0.11.3,<0.14,>=0.11.1 is required by {'transformers'}`

I am not a Python guy, so forgive me if there is an easy fix for this.
Thanks!

**Comment by gidon285** [comment, marked as answer]
change line 31 in setup.py to 
'tokenizers==0.13.2',

**Reply by tricaricom** [reply]
That worked!

Thanks!
